code/modules/mob/living/carbon/human/life/handle_status_effects.dm

```diff
@@ -17,10 +17,10 @@
 	if(command_aura)
 		command_aura_tick--
 
-		if(command_aura_tick < 1)
+		if(command_aura_tick < 1 || IsMute()) //Null the command aura if we're muted or its duration is over
 			command_aura = null
 
-		if(stat == CONSCIOUS)
+		if(stat == CONSCIOUS && !isdeaf()) //Must be conscious and not deaf.
```

Comment:
Remove this change this is a severe orders nerf

Reply:
Changed this to require the marine be both deafened and muted, so only the Silence ability can cancel orders.